src/bilder/images/edxapp_v2/files/docker-compose.yaml.tmpl

```diff
@@ -14,6 +14,8 @@ services:
     volumes:
     - /opt/data:/openedx/data
     - /opt/data/media:/openedx/media
+    - /opt/staticfiles-${ENVIRONMENT_TIER}:/openedx/staticfiles:ro
+    - /opt/staticfiles-${ENVIRONMENT_TIER}/bundles:/openedx/edx-platform/common/static/bundles:ro
```

Comment:
Strictly speaking we don't need to mount the staticfiles into _all_ of these containers/services. Really they should only matter to two web containers. There may be performance gains by not mounting it when it isn't needed.